.env.example

```diff
@@ -1,3 +1,7 @@
 MONGODB_URL_LOCAL = mongodb://127.0.0.1:27017/listcrudnest
 MONGODB_URL_Live = 
-JWT_SECRET = ilovenestjs
```

Comment:
in .env.example never have any data rather than keys

Reply:
Okay